### PR TITLE
Add Draw Sidebar

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Adds a sidebar in the "Draw" stage with the drawing tool.

### Demo

bee-draw-sidebar

### Notes

There's an upcoming PR that gets rid of the Analyze view. With that in place, the "Confirm" button will stop crashing.

I'm putting this up quickly so other people have an idea of what I'm doing so they can start working off of this.

## Testing Instructions

Pull down this branch, run the bundle script, and open [http://localhost:8000](http://localhost:8000). Ensure you are able to draw a shape, cancel drawing, and re-draw after finishing it.

Connects #22

### CODE REVIEW

File: src/icp/js/src/draw/templates/window.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h1>Select Area</h1>
+<p>We are going to make placeholder text great again. Greater than ever before. Look at these words. Are they small words? And he referred to my words - if they're small, something else must be small. I guarantee you there's no problem, I guarantee. An 'extremely credible source' has called my office and told me that Lorem Ipsum's birth certificate is a fraud.</p>
```
Comment (arottersman) [inline src/icp/js/src/draw/templates/window.html]: 🇺🇸

File: src/icp/js/src/draw/templates/window.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{% if not isDrawing and not isDrawn %}
+    <button id="start-drawing" class="btn btn-active btn-lg btn-block" type="button">Start drawing</button>
+{% endif %}
+{% if isDrawing %}
+    <button id="cancel-drawing" class="btn btn-lg btn-block" type="button">Cancel drawing</button>
+{% endif %}
+{% if isDrawn %}
+    <!-- TODO Point to actual Bee Model, not TR-55 -->
+    <a data-url="/project/new/tr-55" class="btn btn-active btn-lg btn-block">Confirm →</a>
+    <button id="cancel-drawing" class="btn btn-lg btn-block" type="button">⃪ Redraw area</button>
```
Comment (rajadain) [inline src/icp/js/src/draw/templates/window.html]: Hmm I should probably replace this non-working ← with an icon or something

**Comment by arottersman** [comment]
+1 Tested -- looks great. Great job on the clean sliding side bar.

File: src/icp/js/src/draw/templates/window.html (hunk 2 above)
Comment (arottersman) [inline src/icp/js/src/draw/templates/window.html]: I figured those were placeholders until we added icons. Also interesting that it doesn't show up here when it renders fine in the browser:
screen shot 2016-11-09 at 4 22 20 pm

Comment (arottersman) [inline src/icp/js/src/draw/templates/window.html]: Wireframe appears to use `fa-arrow-right [&#xf061;]`
screen shot 2016-11-09 at 4 24 34 pm

**Comment by rajadain** [comment]
Added a fixup for the arrows.

**Comment by jtarricone** [comment]
+1 Looks good, the UI goes into the weird error state upon confirming but that's probably expected.

**Comment by rajadain** [comment]
> UI goes into the weird error state upon confirming but that's probably expected.

Yeah, that's fixed in #34.

Thanks for taking a look! Going rebase and merge now ...

**Pull request merged**